first_app/api/comment.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+from first_app.db import db
+from first_app.config import SECRET_KEY
+from .login_required import login_required
+from first_app.models import Post, Comment
+from flask import Flask, g, request, render_template, flash, Blueprint
+
+comment_urls = Blueprint("comment",__name__)
+
+@comment_urls.route("/api/v1/create-comment/<int:user_id>/post/<int:post_id>",
+                    methods=["POST"])
+@login_required
+def create_comment_api(user_id, post_id):
+    data = request.json
+    if user_id != g.user_id:
+        return{"error":"Request data isn't yours"},400
+    author_id = g.user_id
+    comments_post_id = post_id
+    text = data["text"]
+    created = datetime.datetime.now()
+    if not text:    
+        return {"error": "немає text"},400
+    comment = Comment(author_id,
+                    comments_post_id,
+                    created,
+                    text)
+    db.session.add(comment)
+    db.session.commit()    
+    return {"status":"Published"}, 200
+
+@comment_urls.route("/api/v1/post-comments/post/<int:post_id>",
+                    methods=["GET"])
+@login_required
+def post_comments_api(post_id):
+    income_author_id = g.user_id
+    post = Post.query.filter(Post.deleted== False,
+                             Post.id == post_id).first()
+    if not post:
+        return{"error":"Wrong post_id"},400
+    comments_db = Comment.query.filter(Comment.post_id == post_id,
+                                       Comment.deleted == False).all()
+    comments = []
+    
+    for comment in comments_db:
```

Comment:
Maybe instead on manual format comments use https://marshmallow.readthedocs.io/en/stable/. Keep learning with this direction. This is names as serialisation / deserialisation.